documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,56 @@
+== External Trigger on the Global Shutter Camera
+The global shutter camera can be triggered externally by pulsing the XTR pin on the board.
+Multiple cameras can be connected to the same pulse leading to a better way to synchronise two cameras.
+
+The low puslewidth plus an extra 14.26us is equal to the exposure time. i.e. a low pulse of 10000us leads to an exposure time of 10014.26us.
+Framerate is directly controlled by how often you pulse the pin. A PWM frequency of 30Hz will lead to a framerate of 30 frames per second. 
+
+image::images/external_trigger.jpg[Image showing pulse format]
+
+=== Preparation
+
+First, remove/desolder resistor 22 on the GS Camera board. This connects GP0 to XTR and without removal, the camera will not operate
+in external trigger mode.
+
+Next solder a wire to the touchpoint of XTR on the GS Camera board. 
+Connect these to the pico - XTR to any pin. Pin 28 is used in this example. Ground is not needed to be connected.
+==== Boot up the Raspberry Pi with the camera connected.
+
+Enable external triggering through superuser mode:
+[,bash]
+----
+sudo su
+echo 1 > /sys/module/imx296/parameters/trigger_mode
+exit
+----
+
+==== Raspberry Pi Pico Code
+[,python]
+----
+from machine import Pin, PWM
+
+from time import sleep
+
+pwm = PWM(Pin(28))
+
+framerate = 30
+shutter = 6000  # In microseconds
+
+frame_length = 1000000 / framerate
+pwm.freq(framerate)
+
+pwm.duty_u16(int((1 - (shutter - 14) / frame_length) * 65535))
+----
+The low pulsewidth is equal to the shutter time, and the frequency of the PWM equals the framerate.
+
+=== Operation
+
+Run the code on the pico, and run the code for the camera:
+[,bash]
+----
+libcamera-hello -t 0 --qt-preview --shutter 3000
```

Comment:
So there's no option that you need to give to the libcamera apps to tell them that they're being used in "external trigger" mode?